### PR TITLE
ACTIN-934 Parameterize sync scripts to support multiple projects

### PR DESCRIPTION
**Pull request by pauldwolfe**
Syncing (with rsync) the data automatically to these buckets will ensure we don't keep data past the retention period. 

This change parameterizes the locate_files bucket functions such that we can pass actin-nki to the same functions,
keeping actin-prod as default.

### CODE REVIEW

File: actin/operational/sync_actin_data_to_vm [hunk 1]
```diff
@@ -3,21 +3,26 @@
...
-gsutil -m rsync -r $(locate_actin_trial_config_bucket) $(locate_actin_trial_config_directory)
-gsutil -m rsync -r $(locate_actin_ctc_config_bucket) $(locate_actin_ctc_config_directory)
-gsutil -m rsync -r $(locate_actin_prod_data_bucket)/molecular/ $(locate_actin_molecular_directory)
-gsutil -m rsync -r $(locate_actin_prod_data_bucket)/treatment_match/ $(locate_actin_treatment_match_directory)
+gsutil -m rsync -r -d $(locate_actin_clinical_feed_input_bucket $project) $(locate_actin_clinical_input_feed_directory)
+gsutil -m rsync -r -d $(locate_actin_clinical_curation_bucket $project) $(locate_actin_clinical_curation_directory)
+gsutil -m rsync -r -d $(locate_actin_clinical_output_bucket $project) $(locate_actin_curated_clinical_directory)
+gsutil -m rsync -r -d $(locate_actin_trial_config_bucket $project) $(locate_actin_trial_config_directory)
+gsutil -m rsync -r -d $(locate_actin_ctc_config_bucket $project) $(locate_actin_ctc_config_directory)
+echo "gsutil -m rsync -r -d $(locate_actin_prod_data_bucket $project)/molecular/ $(locate_actin_molecular_directory)"
```
Comment (kzuberihmf) [inline actin/operational/sync_actin_data_to_vm:19]: from testing?

Comment (pauldwolfe) [reply actin/operational/sync_actin_data_to_vm:19]: Yeah sorry, mentioned in the shared thread with Simon to not look at this PR yet. I'm going to reorganize it.

File: actin/operational/sync_actin_operations_vm [hunk 2]
```diff
@@ -5,13 +5,27 @@ source locate_files || exit 1
...
+gsutil -m rsync -r -d $(locate_actin_clinical_feed_input_bucket $project) $(locate_actin_clinical_input_feed_directory)
+gsutil -m rsync -r -d $(locate_actin_clinical_curation_bucket $project) $(locate_actin_clinical_curation_directory)
+gsutil -m rsync -r -d $(locate_actin_trial_config_bucket $project) $(locate_actin_trial_config_directory)
+gsutil -m rsync -r -d $(locate_actin_ctc_config_bucket $project) $(locate_actin_ctc_config_directory)
+gsutil -m rsync -r -d $(locate_actin_trial_database_bucket $project) $(locate_actin_trial_database_directory)
+gsutil -m rsync -r -d $(locate_actin_analysis_bucket $project)/molecular/ $(locate_actin_molecular_directory)
+gsutil -m rsync -r -d $(locate_actin_analysis_bucket $project)/treatment_match/ $(locate_actin_treatment_match_directory)
+gsutil -m rsync -r -d $(locate_actin_analysis_bucket $project)/reports/ /data/actin/reports/
+
+info "Syncing ACTIN patient data to GCP"
```
Comment (pauldwolfe) [inline actin/operational/sync_actin_operations_vm:25]: @spdeleeuw Do you know why we do this? I would have expected us to push data to the prod_data_bucket via the archive_after_sharing scripts. I would like to use the directories on the VM as an easy place to keep the latest outputs, but if I keep things this way risk overwriting what's in the share bucket.

TLDR; I'd like to remove this synchronization.

Comment (spdeleeuw) [reply actin/operational/sync_actin_operations_vm:25]: Sorry Paul, but I honestly do not know what this script is used for. It's not part of regular production activities.
